config/config/sc-config.yaml

```
@@ -410,7 +410,15 @@ grafanaLabelEnforcer:
 prometheus:
   retention:
     alertmanager: 72h
-
+  s3BucketAlerts:
+    limit:
+      enabled: false
+      percent: 80
+      sizeQuotaGB: 1000
```

This is maybe also a bit more descriptive
```suggestion
  s3BucketAlerts:
    size:
      enabled: false
      percent: 80
      sizeQuotaGB: 1000
```